### PR TITLE
add hono example

### PR DESCRIPTION
**Pull request by davemooreuws**

### CODE REVIEW

File: nitric-hono/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Simple & secure direct-to-bucket (e.g. S3) file uploads from modern browsers with signed URLs using the [Hono Web Framework](https://hono.dev/) for APIs and the Nitric SDK for bucket and file access.
+
+## Usage
+
+### Step 1: Install Nitric
+
+Follow the steps in the [installation guide](https://nitric.io/docs/installation)
+
+### Step 2: Run your project locally Nitric
```
Comment (HomelessDinosaur) [inline nitric-hono/README.md]:
```suggestion
### Step 2: Run your Nitric project locally
```

File: nitric-hono/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+```bash
+yarn install
+yarn run dev
```
Comment (HomelessDinosaur) [inline nitric-hono/README.md]:
```suggestion
yarn dev
```

File: nitric-hono/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+[Nitric](https://nitric.io) is a framework for rapid development of cloud-native and serverless applications. Define your apps in terms of the resources they need, then write the code for serverless function based APIs, event subscribers and scheduled jobs.
+
+Apps built with Nitric can be deployed to AWS, Azure or Google Cloud all from the same code base so you can focus on your products, not your cloud provider.
+
+Nitric makes it easy to:
+
+- Create smart serverless functions and APIs
+- Build reliable distributed apps that use events and/or queues
+- Securely store, retrieve and rotate secrets
```
Comment (HomelessDinosaur) [inline nitric-hono/README.md]:
```suggestion
- Securely store and retrieve secrets
```

File: nitric-hono/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- Read and write files from buckets
+
+## Documentation
+
+The full documentation is available at [nitric.io/docs](https://nitric.io/docs).
+
+We're completely opensource and encourage [code contributions](https://nitric.io/docs/contributions).
```
Comment (HomelessDinosaur) [inline nitric-hono/README.md]:
```suggestion
We're completely open-source and encourage [code contributions](https://nitric.io/docs/contributions).
```

File: nitric-hono/README.md (hunk 1 above)
Comment (davemooreuws) [reply nitric-hono/README.md]: @HomelessDinosaur Copied the README from another example. Will apply these to every example readme in another commit.

**Review by HomelessDinosaur — APPROVED** [review]

**Pull request merged**